CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changes to the Mapbox Directions SDK for iOS
 
+## v0.25.0
+
+* Added `Directions.fetchAvailableOfflineVersions(completionHandler:)` for listing available offline versions. ([#303](https://github.com/mapbox/MapboxDirections.swift/pull/303))
+* Added `Directions.downloadTiles(in:version:session:completionHandler:)` for downloading a tile pack. ([#303](https://github.com/mapbox/MapboxDirections.swift/pull/303))
```

Comment:
Does docs/jazzy.yml need to be updated to reflect any changes to top-level types in #303?

Reply:
Seems to be working b/c it's just an extension on Directions.

![screenshot 2018-11-01 18 02 38](https://user-images.githubusercontent.com/764476/47866913-590a1780-de00-11e8-856b-9d40e91c2792.png)